### ISSUE TITLE
Fragments error: Unexpected name __typename

### ISSUE THREAD

**Issue by richburdon** [jimkyndemeyer/js-graphql-intellij-plugin#57]
Using fragments I get redline warnings for referenced types.

NOTE: the fragments don't explicitly declare __typename.

```
import gql from 'graphql-tag';

export const ItemFragment = gql`
  fragment ItemFragment on Item {
    bucket
    id
    type
    labels
    title
    description
  }
`;

const ContactQuery = gql`
  query ContactQuery($itemId: ID!) { 
    
    item(itemId: $itemId) {
      ...ItemFragment
      ...ContactFragment
    }
  }

  ${ItemFragment}  <==== Syntax Error GraphQL(58:3) Unexpected Name "__typename"
  ${ContactFragment}  
`;
```

**Comment by jimkyndemeyer** [comment]
Hi Rich.

It looks like this is the same as #44 which is fixed in the just released 1.5.0 of the plugin.

**Comment by jimkyndemeyer** [comment]
Closing this. Feel free to re-open if not fixed with 1.5.0 and #44.
